test/runScript-mock-pMap.spec.js

```diff
@@ -32,7 +33,7 @@ describe('runScript', () => {
     try {
       await res[0].task()
     } catch (err) {
-      expect(err.message).toMatch(`🚫 test got an unexpected error.
+      expect(err.message).toMatch(`${logSymbols.error} test got an unexpected error.
```

Comment:
Should we switch to http://facebook.github.io/jest/docs/en/expect.html#tothrowerrormatchingsnapshot instead

Reply:
agree! do it in this PR or on separate PR?

Reply:
Just a note, there are some `jest` related things which aren't optimum. Example - https://github.com/okonet/lint-staged/blob/33a7a4ae6a89603eb929ffe70e3900021a78f307/test/__mocks__/npm-which.js#L1-L14
This isn't using `jest.genMockFromModule` correctly. 

Docs - http://facebook.github.io/jest/docs/en/jest-object.html#jestgenmockfrommodulemodulename
Guide - http://facebook.github.io/jest/docs/en/manual-mocks.html

Another example for less optimal assertion - https://github.com/okonet/lint-staged/blob/33a7a4ae6a89603eb929ffe70e3900021a78f307/test/runScript-mock-pMap.spec.js#L21-L25

Reply:
@luftywiranda13 doesn't matter to me TBH.

@sudo-suhas could you help with those examples and create a PR that would "fix" them before this one is merged?

Reply:
I am a bit swamped at the moment. Trying to finish something at work. I can do it tomorrow. I think I'd prefer optimising jest related things in a separate PR after all the changes I have in the pipeline:

- eslint-plugin-lodash
- fix failing tests on windows
- rebase dedent PR
- `getConfig` - `_normalized` property
- try pify

Reply:
+1 on pify 👍